examples/QmlBarcodeReader/qml/ScannerPage.qml

```diff
@@ -7,6 +7,8 @@ import com.scythestudio.sbarcodereader 1.0
 ApplicationWindow {
   id: root
   visible: true
+  height: if(Qt.platform.os != "android") camera.viewfinder.resolution.height
```

Comment:
```suggestion
  height: Qt.platform.os == "android" || Qt.platform.os == "ios" ? Screen.height: camera.viewfinder.resolution.height
```

Your solution was bad because of two reasons:
1. You missed case for other mobile OS - iOS
2. If condition was false, then height and width were set to undefined